### PR TITLE
Fix(web-react): Unsupported export in client boundary #DS-1549

### PR DESCRIPTION
**Pull request by curdaj**
### Issue reference

- [x] delete demo files in next app

### CODE REVIEW

File: examples/next-with-app-router/src/app/page.tsx
Comment (curdaj) [inline examples/next-with-app-router/src/app/page.tsx:1]: Will be deleted - only for Demo

File: examples/next-with-app-router/src/app/testSsrComponent.tsx
Comment (curdaj) [inline examples/next-with-app-router/src/app/testSsrComponent.tsx:1]: Will be deleted - only for Demo

File: packages/web-react/src/components/Flex/index.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 'use client';
 
-export * from './Flex';
-export * from './useFlexStyleProps';
```
Comment (literat) [inline packages/web-react/src/components/Flex/index.ts:4]: **note (if-minor):** When there are only [named exports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) the `*` asterisk is ok here. So when you add a new function and export it, you do not need to write it here like you do it in your modification. So I think for the hook it was ok.

**Comment by literat** [comment]
Please, use the integration branch as a target branch.

**Review by literat — APPROVED** [review]
Looking good 👍

**Comment by literat** [comment]
Tested on the Cyborg and it is ok.

**Pull request merged**